testing_and_setup/compass/ocean/jigsaw_to_MPAS/build_mesh.py

```diff
@@ -53,17 +56,14 @@ def build_mesh(
         da.to_netcdf(cw_filename)
         plot_cellWidth=True
         if plot_cellWidth:
-            import matplotlib
-            from cartopy import config
-            import cartopy.crs as ccrs
-            matplotlib.use('Agg')
             fig = plt.figure()
             fig.set_size_inches(16.0, 8.0)
             plt.clf()
             ax = plt.axes(projection=ccrs.PlateCarree())
             ax.set_global()
-            im = ax.imshow(cellWidth, origin='lower', transform=ccrs.PlateCarree(
-            ), extent=[-180, 180, -90, 90], cmap='jet')
+            im = ax.imshow(cellWidth, origin='lower',
+                           transform=ccrs.PlateCarree(),
+                           extent=[-180, 180, -90, 90], cmap='viridis_r')
```

Comment:
Make this `rainbow`.

Reply:
Please don't-- this doesn't add value and rainbow is a notoriously bad colormap.

Reply:
(unless rainbow is the perceptually uniform one, not "jet")  In that case this is ok.

Reply:
@pwolfram, thanks for the feedback.  I agree.  How about another solution?  I have been using a color map from paraview for resolution that is made up of 3 perceptually uniform segments ("Blue - Green - Orange").  It will be more complicated to add this to COMPASS, since it's not standard in matplotlib, but I'll see what I can do.  Would that work for you?  @mark-petersen, what do you think?

Reply:
The problem with `viridis` is that it's too dark and the black outlines from `cartopy` don't show up well, but this could be fixed by drawing in the continents, not just the outlines.

Reply:
I'm happy with perceptually uniform solutions, but don't like it when you can see "lines" like yellow, in the colormap.   But, I'm ok with non-jet solutions like proposed below-- thanks!